### PR TITLE
Updated usage example in the README.md file

### PR DESCRIPTION
**Pull request by prassee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,29 +43,45 @@ that describes your build. Here is an example
...
+Dependencies could also be declared using SBT style.
+
+```scala
+class Build(val context: cbt.Context) extends PackageJars {
+...
+// sbt compatible dependencies definition
+override def dependencies =
+  super.dependencies ++
+    Resolver(mavenCentral).bind("com.typesafe.play" %% "play-json" % "2.4.4",
+      "joda-time", "joda-time", "2.9.2")
```
Comment (cvogt) [inline README.md]: I personally prefer line breaks after ( and after , and before ) if the line is too long.

File: README.md [hunk 2]
```diff
@@ -43,29 +43,45 @@ that describes your build. Here is an example
...
-      ScalaDependency("com.typesafe.play", "play-json", "2.4.4"),
-      MavenDependency("joda-time", "joda-time", "2.9.2")
-      // also supported for SBT syntax compatibility:
-      // "com.typesafe.play" %% "play-json" % "2.4.4"
-      // Maven.central % "joda-time" % "joda-time % "2.9.2"
-    )
+      Resolver(mavenCentral).bind(
+        // encouraged way to declare dependencies
+        ScalaDependency("com.typesafe.play", "play-json", "2.4.4"),
+        MavenDependency("joda-time", "joda-time", "2.9.2"))
```
Comment (cvogt) [inline README.md]: could you break the line between the two )) ?

File: README.md (hunk 1 above)
Comment (cvogt) [inline README.md]: there is a wrong comma between the "joda-time"s

File: README.md [hunk 3]
```diff
@@ -43,29 +43,45 @@ that describes your build. Here is an example
...
-class Build(val context: cbt.Context) extends Publish{
-  override def version = "0.6.1"
+class Build(val context: cbt.Context) extends PackageJars {
+  override def defaultVersion = "0.6.1"
+
+  override def name = "play-json-extensions"
+
   override def groupId = "org.cvogt"
+
   override def artifactId = "play-json-extensions"
```
Comment (cvogt) [inline README.md]: this one can be left out btw. It's taken from `name`

File: README.md (hunk 1 above)
Comment (prassee) [inline README.md]: addressed

**Pull request merged**